src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs

```diff
@@ -210,6 +224,29 @@ public virtual byte[] Decrypt(byte[] ciphertext, byte[] authenticatedData, byte[
             }
         }
 
+        /// <summary>
+        /// Calls <see cref="Dispose(bool)"/> and <see cref="GC.SuppressFinalize"/>
+        /// </summary>
+        public void Dispose()
```

Comment:
Are usages of AuthenticatedEncryptionProvider encapsulated in using statements? 
If not, shouldn't there be?

Reply:
I will add that, thanks.

Reply:
resolved by: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/a62d951d0c2a5759d03372b404f345b333fcb438